pkgs/development/gnuradio-modules/osmosdr/default.nix

```
@@ -27,24 +29,30 @@ let
   version = {
     "3.7" = "0.1.5";
     "3.8" = "0.2.3";
```

```suggestion
    "3.8" = "0.2.4";
```
The new version is [compatible with gnuradio 3.8](https://gitea.osmocom.org/sdr/gr-osmosdr/src/commit/82d6b6db78c829ddf7511826bd2b64be2bc189d5/CMakeLists.txt#L23) too.

---

After doing that (and updating the hash on a line below), the build failed:

```
$ nix-build -A gnuradio3_8Packages.osmosdr
[...]
CMake Error at CMakeLists.txt:45 (find_package):
  Could not find a configuration file for package "Gnuradio" that is
  compatible with requested version "3.9".

  The following configuration files were considered but not accepted:

    /nix/store/h8aic81cmz7mzfdwzp3xkrrcg2a99fcg-gnuradio-3.8.5.0/lib/cmake/gnuradio/GnuradioConfig.cmake, version: 3.8.5.0



-- Configuring incomplete, errors occurred!
See also "/build/gr-osmosdr/build/CMakeFiles/CMakeOutput.log".
error: build of '/nix/store/r9md502855k6j39lj4314qfhh246fg2i-gr-osmosdr-0.2.4.drv' failed [...]
```

Btw, you linked to [this](https://gitea.osmocom.org/sdr/gr-osmosdr/src/commit/82d6b6db78c829ddf7511826bd2b64be2bc189d5/CMakeLists.txt#L23), which is the minimum *cmake* version, not gnuradio :-)

---

> After doing that (and updating the hash on a line below), the build failed:
> 
> ```
> $ nix-build -A gnuradio3_8Packages.osmosdr
> [...]
> CMake Error at CMakeLists.txt:45 (find_package):
>   Could not find a configuration file for package "Gnuradio" that is
>   compatible with requested version "3.9".
> 
>   The following configuration files were considered but not accepted:
> 
>     /nix/store/h8aic81cmz7mzfdwzp3xkrrcg2a99fcg-gnuradio-3.8.5.0/lib/cmake/gnuradio/GnuradioConfig.cmake, version: 3.8.5.0
> 
> 
> 
> -- Configuring incomplete, errors occurred!
> See also "/build/gr-osmosdr/build/CMakeFiles/CMakeOutput.log".
> error: build of '/nix/store/r9md502855k6j39lj4314qfhh246fg2i-gr-osmosdr-0.2.4.drv' failed [...]
> ```
> 
> Btw, you linked to [this](https://gitea.osmocom.org/sdr/gr-osmosdr/src/commit/82d6b6db78c829ddf7511826bd2b64be2bc189d5/CMakeLists.txt#L23), which is the minimum _cmake_ version, not gnuradio :-)

I am sorry, I am absolutely blind :smile: